security/libgcrypt-tests.py

```diff
@@ -34,33 +34,49 @@ def setUp(self):
         # Check for basic utilities
         smm = SoftwareManager()
         detected_distro = distro.detect()
-        deps = ['gcc', 'make']
-        if detected_distro.name in ['rhel', 'SuSE', 'fedora', 'centos']:
-            deps.extend(["autoconf", "automake", "libgpg-error-devel",
-                         "libgpg-error", "transfig"])
-        else:
+        deps = ['gcc', 'make', "autoconf", "automake", "libgpg-error-devel",
+                "libgpg-error"]
+        if detected_distro.name in ['rhel', 'fedora', 'centos']:
+            deps.extend(["transfig"])
+        elif 'Ubuntu' in detected_distro.name:
```

Comment:
@nasastry what about sles here?
isn't supported for this feature? if not, we can add as unsupported

Reply:
@Naresh-ibm Already attached SLES related output in the above.
RHEL & SLES are supported.
It's just that SLES doesn't require any specific package as pre-requisite.